test/e2e/flowaggregator_test.go

```diff
@@ -574,6 +569,32 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records: ", len(recordSlices))
 }
 
+func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcPodName string, srcIP string, dstIP string, dstPort int32, isIPv6 bool) {
+	timeStart := time.Now()
+	var cmd string
+	if !isIPv6 {
+		cmd = fmt.Sprintf("wget -O- %s:%d", dstIP, dstPort)
+	} else {
+		cmd = fmt.Sprintf("wget -O- %s", dstIP)
+	}
```

Comment:
I don't know if this is specific to this test but why don't we need the port for the IPv6 version? Can we add a comment?

Reply:
I found if added the port, the command will fail on ipv6 cluster. I'll add a comment.

Reply:
Sorry I forgot to include `[]` around the IP when adding the port, so I thought adding port will fail. Let me test again.